test/WebJobs.Script.Tests.Integration/Management/MeshServiceClientTests.cs

```diff
@@ -209,5 +209,77 @@ public async Task CreatesBindMount()
                 ItExpr.Is<HttpRequestMessage>(r => IsCreateBindMountRequest(r, sourcePath, targetPath)),
                 ItExpr.IsAny<CancellationToken>());
         }
+
+        [Fact]
+        public async Task UsesCustomJsonSerializer()
```

Comment:
This test really isn't testing the actual production code change you're making (that when no serializer specified, we're using a set of explicit settings rather than taking global defaults. To do so, you'd use a try/finally block, and have the test repro the actual issue by capturing the current global settings, modifying them for the test to change enum behavior, and verify those changes have no effect. Then restore the previous global settings in the finally block.

You can then remove the client ctor serializer param.